content/docs/chat_basics/public-rooms/_index.md

```diff
@@ -57,6 +57,77 @@ Click on "Join the discussion": you can now participate in the room!
 
 ![](room-participate.png)
 
+## Hiding images by default
+
+In public rooms, sometimes people post inappropriate content, including images
+you don't want to see. Hopefully a room's moderators will tidy things up quite
+quickly, but many people prefer to make images in public rooms hidden, allowing
+them to show individual images when they are ready.
+
```

Comment:
maybe it would be worthwhile to add some anchor links here to jump to the client that interests me?

Reply:
I don't see any anchor links in the rest of the site, so I don't feel particularly ready to add a new style here. I don't think it's too hard to navigate?